Documentation/StaticFrameworks.md

```diff
@@ -0,0 +1,55 @@
+# Build static frameworks to speed up your app’s launch times
+Carthage supports building static frameworks in place of dynamic libraries when used in concert with Keith Smiley’s `ld.py` script, published [here](https://github.com/keith/swift-staticlibs/blob/master/ld.py). If you have many dynamic frameworks, you may have noticed that your application launch times can be quite slow relative to other applications. To alleviate this slowdown, Apple suggests that you embed [at most six dynamic frameworks](https://developer.apple.com/videos/play/wwdc2016/406/?time=1794) into your applications. Unfortunately, Xcode has not supported building static Swift libraries out of the box since Apple made that recommendation, so it is a bit tricky to follow this advice. The goal of this guide is to show you how to reduce the number of embedded dynamic libraries in your application with some simple wrappers around Carthage.
+
```

Comment:
Can you add a caveat paragraph here that explains that static frameworks aren't officially supported by Apple?

It'd probably also be good to say that these aren't officially supported by Carthage either, or something like that. I'm not quite sure what the language should be.

Basically: you should only do this if you know what you're doing and are willing to take some responsibility for it.